ray-operator/main.go

```diff
@@ -92,7 +92,7 @@ func main() {
 		"Encoder to use for logging stdout. Valid values are 'json' and 'console'. Defaults to 'json'")
 	flag.BoolVar(&enableBatchScheduler, "enable-batch-scheduler", false,
 		"(Deprecated) Enable batch scheduler. Currently is volcano, which supports gang scheduler policy.")
```

Comment:
Can we add a note in this flag "Use --batch-scheduler instead"

Reply:
Thanks. Fixed.